### PR TITLE
Fixing a typo

### PR DESCRIPTION
**Pull request by DeityLink**

### CODE REVIEW

File: code/game/gamemodes/cult/ritual.dm [hunk 1]
```diff
@@ -373,7 +373,7 @@ var/global/list/rune_list = list() // HOLY FUCK WHY ARE WE LOOPING THROUGH THE W
 		if(M.invisibility != 0)
 			M.invisibility = 0
 			user.visible_message(
-				"<span class='warning'>[user] drags the ghost to our plan of reality!</span>",
+				"<span class='warning'>[user] drags the ghost to our plane of reality!</span>",
 				"<span class='warning'>You drag the ghost to our plan of reality!</span>"
```
Comment (Exxion) [inline code/game/gamemodes/cult/ritual.dm:377]: henk

Comment (SysadminJeroen) [inline code/game/gamemodes/cult/ritual.dm:377]: Did you really forget this one?

Comment (9600bauds) [inline code/game/gamemodes/cult/ritual.dm:377]: Flan of reality microwave recipe when?

Comment (SysadminJeroen) [inline code/game/gamemodes/cult/ritual.dm:377]: @9600bauds: I might make it if no one else does.

**Pull request merged**